js/reference.js

```diff
@@ -2189,15 +2189,27 @@
 
             // this function will take care of adding column and asset column
             var addColumn = function (col) {
-                if(col.type.name === "text" && col.annotations.contains(module._annotations.ASSET)) {
+                if (col.type.name === "text" && col.annotations.contains(module._annotations.ASSET)) {
+
                     if (("url_pattern" in col.annotations.get(module._annotations.ASSET).content)) {
-                        // add asset annotation
-                        var assetCol = new AssetPseudoColumn(self, col);
-                        assetColumns.push(assetCol);
-                        self._referenceColumns.push(assetCol);
-                        return;
+                        
+                        var urlPattern = col.annotations.get(module._annotations.ASSET).content.url_pattern;
+                        
+                        // If url_pattenr doesn't has hatrac in it then consider the column as a plain text column and proceed
+                        if ((typeof urlPattern !== 'string') || (urlPattern.indexOf('/hatrac') === -1)) {
```

Comment:
Please use `console.debug` instead, and there are some grammatical errors. (doesn't has/ doesn't contains)

Reply:
Node.js has no `console.debug` method, so I used console.log instead.

Reply:
oh, I didn't know that.